internal/cli/backup/restores/start.go

```diff
@@ -174,7 +174,7 @@ func (opts *StartOpts) checkIsFlexCluster() error {
 		return nil
 	}
 
-	if !atlasv2.IsErrorCode(err, cannotUseFlexWithClusterApisErrorCode) {
+	if !atlasClustersPinned.IsErrorCode(err, cannotUseFlexWithClusterApisErrorCode) {
```

Comment:
[q] do we also need to check `	if !atlasv2.IsErrorCode(err, cannotUseFlexWithClusterApisErrorCode) {` in an && statement?

Reply:
no, the error is coming only from `atlasClustersPinned`. The issue was that the error was coming from SDK `atlasClustersPinned` but we were checking the error with `atlasv2` with is a different sdk version